src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateServiceTest.kt

```diff
@@ -352,7 +352,90 @@ class ReleaseDateServiceTest {
         conditionalReleaseDate = null,
       )
 
-      assertThat(service.isInHardStopPeriod(licence, now)).isFalse()
+      assertThat(service.isInHardStopPeriod(licence, now)).isFalse
+    }
+  }
+
+  @Nested
+  inner class `Licence is due to be released in the next two days` {
+    @Test
+    fun `licence is not due to be released in the next two days if there is no release date`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = null,
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isFalse
+    }
+
+    @Test
+    fun `licence is not due to be released in the next two working days when CRD is yesterday`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock).minusDays(1),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isFalse
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is today`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isTrue
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is 1 day in the future`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock).plusDays(1),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isTrue
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is 2 working days in the future`() {
+      val licence = createCrdLicence().copy(
+        actualReleaseDate = null,
+        conditionalReleaseDate = LocalDate.now(clock).plusDays(3),
+      )
+
+      assertThat(service.isDueToBeReleasedInTheNextTwoWorkingDays(licence)).isTrue
+    }
+
+    @Test
+    fun `licence is due to be released in the next two working days when CRD is more than 2 working days in the future`() {
```

Comment:
Minor, not worth its own commit, but
`licence is **not** due to be released ...`

Reply:
👍  fixed this